ang/civicase/case/details/file-tab/directives/case-details-file-tab.directive.js

```diff
@@ -9,17 +9,154 @@
         item: '=civicaseCaseDetailsFileTab',
         refresh: '=?refreshCallback'
       },
-      controller: civicaseCaseDetailsFileTabController
+      controller: 'civicaseCaseDetailsFileTabController'
     };
+  });
+
+  module.controller('civicaseCaseDetailsFileTabController', civicaseCaseDetailsFileTabController);
+
+  /**
+   * Controller function for the directive
+   *
+   * @param {object} $scope controllers scope object
+   * @param {object} BulkActions bulk actions service
+   * @param {object} crmApi service to call civicrm backend
+   * @param {Function} formatActivity format activity service
+   */
+  function civicaseCaseDetailsFileTabController ($scope, BulkActions, crmApi, formatActivity) {
+    $scope.ts = CRM.ts('civicase');
```

Comment:
there is a global `ts` function already injected to all scopes so there is no need to inject it for this particular scope.

More info here:
https://github.com/compucorp/uk.co.compucorp.civicase/pull/467/files#diff-8de0fcae9b87624e12434a30f5997304R23

If you come across a directive that is injecting `$scope.ts` manually, please fix it.